### PR TITLE
Various coding style cleanup

### PR DESCRIPTION
**Pull request by rxin**
Merge #205 before this one.

### CODE REVIEW

File: src/main/scala/shark/execution/optimization/ColumnPruner.scala [hunk 1]
```diff
@@ -38,11 +54,13 @@ class ColumnPruner(@transient op: TopOperator[_], @transient tbl: Table) extends
   /**
    * Computes the column names that are referenced in the Query
    */
-  private def computeColumnsToKeep(op: Operator[_],
-    cols: HashSet[String], parentOp: Operator[_] = null): Unit = {
+  private def computeColumnsToKeep(
+      op: Operator[_], cols: HashSet[String], parentOp: Operator[_] = null): Unit = {
```
Comment (harveyfeng) [inline src/main/scala/shark/execution/optimization/ColumnPruner.scala]: Each of the constructor params should have its own line.

File: src/main/scala/shark/execution/LateralViewJoinOperator.scala [hunk 2]
```diff
@@ -26,7 +25,6 @@ import scala.reflect.BeanProperty
 
 import org.apache.commons.codec.binary.Base64
 import org.apache.hadoop.hive.ql.exec.{ExprNodeEvaluator, ExprNodeEvaluatorFactory}
-import org.apache.hadoop.hive.ql.exec.{LateralViewJoinOperator => HiveLateralViewJoinOperator}
 import org.apache.hadoop.hive.ql.plan.SelectDesc
 import org.apache.hadoop.hive.ql.plan.LateralViewJoinDesc
```
Comment (harveyfeng) [inline src/main/scala/shark/execution/LateralViewJoinOperator.scala]: ordering

File: src/main/scala/shark/execution/MapSplitPruning.scala [hunk 3]
```diff
@@ -29,8 +27,6 @@ import org.apache.hadoop.hive.ql.udf.generic.GenericUDFOPLessThan
 import org.apache.hadoop.hive.ql.udf.generic.GenericUDFOPEqualOrGreaterThan
 import org.apache.hadoop.hive.ql.udf.generic.GenericUDFOPEqualOrLessThan
 import org.apache.hadoop.hive.ql.udf.generic.GenericUDFIn
-import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspector
```
Comment (harveyfeng) [inline src/main/scala/shark/execution/MapSplitPruning.scala:32]: The GenericUDFOP\* are out of order...

Comment (harveyfeng) [inline src/main/scala/shark/execution/MapSplitPruning.scala:32]: and GenericUDFIn

File: src/main/scala/shark/execution/MemoryStoreSinkOperator.scala [hunk 4]
```diff
@@ -158,10 +158,10 @@ class MemoryStoreSinkOperator extends TerminalOperator {
 
       outputRDD.persist(storageLevel)
 
-      var queryOutputRDD = outputRDD
+      val queryOutputRDD = outputRDD
       if (useUnionRDD) {
         // Handle an INSERT INTO command.
-        var previousRDDOpt: Option[RDD[TablePartition]] =
+        val previousRDDOpt: Option[RDD[TablePartition]] =
           if (isHivePartitioned) {
```
Comment (harveyfeng) [inline src/main/scala/shark/execution/MemoryStoreSinkOperator.scala]: This `if (...)` could be brought up to the line for the `val previousRDDOpt` declaration.

File: src/main/scala/shark/execution/ScriptOperator.scala [hunk 5]
```diff
@@ -29,13 +29,11 @@ import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.hive.conf.HiveConf
 import org.apache.hadoop.hive.ql.exec.{ScriptOperator => HiveScriptOperator}
 import org.apache.hadoop.hive.ql.exec.{RecordReader, RecordWriter, ScriptOperatorHelper}
```
Comment (harveyfeng) [inline src/main/scala/shark/execution/ScriptOperator.scala]: Maybe a `{ScriptOperatorHelper => HiveScriptOperatorHelper}` for consistency?

File: src/main/scala/shark/execution/UnionOperator.scala [hunk 6]
```diff
@@ -19,16 +19,13 @@ package shark.execution
 
 import java.util.{ArrayList, List => JavaList}
 
-import scala.collection.mutable.ArrayBuffer
 import scala.collection.JavaConversions._
 import scala.reflect.BeanProperty
 
-import org.apache.hadoop.hive.ql.exec.{UnionOperator => HiveUnionOperator}
 import org.apache.hadoop.hive.ql.udf.generic.GenericUDFUtils.ReturnObjectInspectorResolver
 import org.apache.hadoop.hive.ql.plan.UnionDesc
```
Comment (harveyfeng) [inline src/main/scala/shark/execution/UnionOperator.scala:25]: plan <=> udf imports should be swapped

File: src/main/scala/shark/SharkEnv.scala [hunk 7]
```diff
@@ -19,10 +19,9 @@ package shark
 
 import scala.collection.mutable.{HashMap, HashSet}
 
-import org.apache.spark.{SparkContext, SparkEnv}
+import org.apache.spark.SparkContext
 import org.apache.spark.rdd.RDD
 import org.apache.spark.scheduler.StatsReportListener
-import org.apache.spark.serializer.{KryoSerializer => SparkKryoSerializer}
```
Comment (harveyfeng) [inline src/main/scala/shark/SharkEnv.scala:25]: The SparkContext import should be last

File: src/main/scala/shark/execution/MapSplitPruning.scala (hunk 3 above)
Comment (harveyfeng) [inline src/main/scala/shark/execution/MapSplitPruning.scala:32]: Actually, its just the GenericUDFOPOr and GenericUDFIn :S

File: src/main/scala/shark/SharkEnv.scala (hunk 7 above)
Comment (harveyfeng) [inline src/main/scala/shark/SharkEnv.scala:25]: Nvm, this one is good...

**Comment by rxin** [comment]
Ok updated.

File: src/main/scala/shark/execution/ScriptOperator.scala [hunk 8]
```diff
@@ -28,14 +28,13 @@ import scala.reflect.BeanProperty
 import org.apache.hadoop.conf.Configuration
 import org.apache.hadoop.hive.conf.HiveConf
 import org.apache.hadoop.hive.ql.exec.{ScriptOperator => HiveScriptOperator}
-import org.apache.hadoop.hive.ql.exec.{RecordReader, RecordWriter, ScriptOperatorHelper}
-import org.apache.hadoop.hive.ql.metadata.HiveException
+import org.apache.hadoop.hive.ql.exec.{RecordReader, RecordWriter}
+import org.apache.hadoop.hive.ql.exec.{ScriptOperatorHelper => HiveScriptOperatorHelper}
```
Comment (harveyfeng) [inline src/main/scala/shark/execution/ScriptOperator.scala:32]: The ScriptOperator and ScriptOperatorHelper imports seem out of order...
Flip the RecordReader and ScriptOperator?

**Comment by harveyfeng** [comment]
LGTM!

**Pull request merged**